examples/conf/vlcs_diva_mldg_dial.yaml

```diff
@@ -5,7 +5,9 @@ val_threshold: 0.8                  # threashold before which training does not
 model: dann_diva                    # combine model DANN with DIVA
 epos: 1                             # number of epochs
 trainer: mldg_dial                  # combine trainer MLDG and DIAL
-gamma_reg: 1.0                      # hyperparameter of DANN
+gamma_reg: 
+  default: 1.0
+  dann: 1.5     
```

Comment:
@MatteoWohlrapp , how about mldg?

Reply:
is this behavior tested somewhere? in which test, for instance, now the gamma_reg value for mldg should be 1.0?  @MatteoWohlrapp

Reply:
i made a pull request to test his, still under work @MatteoWohlrapp 

https://github.com/marrlab/DomainLab/pull/847